packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Mocks/MockHelpers.cs

```diff
@@ -16,14 +16,16 @@ namespace Microsoft.Generator.CSharp.ClientModel.Tests
     internal static class MockHelpers
     {
         public const string MocksFolder = "Mocks";
+        public static string ConfigFilePath = Path.Combine(AppContext.BaseDirectory, MocksFolder);
```

Comment:
shouldn't have public fields we can change this to a property and mark it as readonly